pkg/controller/elasticsearch/validation/webhook.go

```diff
@@ -76,13 +79,21 @@ func (wh *validatingWebhook) validateUpdate(prev esv1.Elasticsearch, curr esv1.E
 	return ValidateElasticsearch(curr, wh.exposedNodeLabels)
 }
 
+// Handle is called when any request is sent to the webhook, satisfying the admission.Handler interface.
 func (wh *validatingWebhook) Handle(_ context.Context, req admission.Request) admission.Response {
 	es := &esv1.Elasticsearch{}
 	err := wh.decoder.DecodeRaw(req.Object, es)
 	if err != nil {
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	// If this Elasticsearch instance is not within the set of managed namespaces
+	// for this operator ignore this request.
+	if !wh.managedNamespaces.Has(es.Namespace) {
```

Comment:
I realise we still need the length check with `Count` on `managedNamespaces` here to cover the case where all namespaces are managed by the operator and `managedNamespaces` is the empty set.

Reply:
thank you.   I've added this to both this file, and the shared webhook validation.